crates/anvil/src/eth/fees.rs

```diff
@@ -259,6 +259,12 @@ impl FeeHistoryService {
                                 .min(U256::from(t.max_fee_per_gas).saturating_sub(base_fee))
                                 .to::<u64>()
                         }
+                        // TODO: This probably needs to be extended to extract 4844 info.
```

Comment:
Will be done in a subsequent PR which modifies `FeeHistory` to accomodate 4844 changes

Reply:
we should add a follow up issue for this to track

Reply:
This should be go on #7242

Reply:
saw it there